packages/metrics-discovery/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-alpine3.20
+
+COPY . /river
+WORKDIR /river
+
+RUN yarn install
+RUN yarn build:metrics-discovery
+RUN corepack enable
```

Comment:
I thought you'd need to enable corepack before using yarn

Reply:
i think it worked because yarn came built in with the image. and if you're curious about why i'm even running 1corepack enable1 if not to install the package managers in the first place - it's because some error message told me to :) 

i'm hoisting it up though 👍